sql/moz-fx-data-shared-prod/telemetry_derived/desktop_funnel_activation_day_6_v1/query.sql

```diff
@@ -1,37 +1,31 @@
 WITH pop AS (
   SELECT
-    ROW_NUMBER() OVER (PARTITION BY client_id) AS rn,
```

Comment:
I believe this is no longer necessary as CFS v2 should always be one row per client?